tests/apps/resiliencyapp/app.go

```diff
@@ -56,12 +64,20 @@ type PubsubResponse struct {
 	Message string `json:"message,omitempty"`
 }
 
-var (
-	daprClient   runtimev1pb.DaprClient
-	callTracking map[string][]CallRecord
-)
-
-var httpClient = utils.NewHTTPClient()
+func init() {
+	p := os.Getenv("DAPR_HTTP_PORT")
+	if p != "" && p != "0" {
+		daprHttpPort, _ = strconv.Atoi(p)
```

Comment:
even in test should we just default on error instead of ignoring it?

Reply:
This env var was added to allow running the tests on a different port and it's only used during development.